### PR TITLE
feat: choosing your signer, light account, modular account doc update

### PR DESCRIPTION
**Pull request by denniswon**
### Detailed summary
- Added a link to the account factory's code in `getAccountInitCode` documentation.
- Updated links in the `introduction.md` file.
- Updated content in the `choosing-a-smart-account.md` file.
- Updated content in the `modular-account.md` file.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by avasisht23 — CHANGES_REQUESTED** [review]
let's keep the intro/overview docs as a separate item. 

this was feedback from Will that it's unintuitive to click on the section header and expect a page, vs. having it be more explicit and visible as a standalone link with something like "Overview"

File: site/smart-accounts/accounts/light-account.md
Comment (denniswon) [inline site/smart-accounts/accounts/light-account.md:1]: This file is already reviewed.

**Comment by denniswon** [comment]
@avasisht23 separated the pr with https://github.com/alchemyplatform/aa-sdk/pull/217

I will accommodate your comment above on that pr.

**Review by avasisht23 — APPROVED** [review]
just relint the config.ts folder. missing an EOF newline.

**Pull request merged**